shelley/chain-and-ledger/formal-spec/delegation.tex

```diff
@@ -954,13 +955,16 @@ \subsection{Delegation and Pool Combined Rules}
     \inference[Seq-delg-ind]
     {
       \var{c}\in\DCertDeleg \Rightarrow \fun{dpool}~{c} \in \dom \var{stpools} \\
-        ptr \leteq (\var{slot},~\var{txIx},~\mathsf{len}~\Gamma - 1) \\~\\
+      ptr \leteq (\var{slot},~\var{txIx},~\mathsf{len}~\Gamma - 1) \\~\\
+      % TODO should that be "len \Gamma" ? at this point \Gamma could be the
```

Comment:
in the spec there is a case distinction separating the empty list, which is handled by `Seq-delg-base`. I'm not sure what is the best thing to do in the STS framework where the case split doesn't happen here.

Reply:
yes, but here we construct the `ptr` which is used in the `DELPL` rule which takes a single certificate `c`. So wouldn't that also compute `len Γ -1` in the case of `∅; c` ?

Reply:
ah yes, the full list is `Gamma;c`, not `Gamma`, you are correct.

Reply:
ok, I'll change it then in the formal spec

Reply:
thank you!